diff --git a/openstack_auth/backend.py b/openstack_auth/backend.py
index c8acd21..8690c37 100644
--- a/usr/lib/python3.6/site-packages/openstack_auth/backend.py
+++ b/usr/lib/python3.6/site-packages/openstack_auth/backend.py
@@ -149,7 +149,7 @@ class KeystoneBackend(object):
             scoped_auth = domain_auth
             scoped_auth_ref = domain_auth_ref
         elif not scoped_auth_ref and not domain_auth_ref:
-            msg = _('You are not authorized for any projects or domains. Please visit https://access.nrec.no first')
+            msg = _('You are not authorized for any projects or domains.')
             raise exceptions.KeystoneNoProjectsException(msg)

         # Check expiry for our new scoped token.
