--- /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py   2017-06-01 16:09:48.000000000 +0200
+++ /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py  2017-08-31 14:13:53.816612322 +0200
@@ -216,7 +216,7 @@
 # of vm booted from image with attached devices
 MIN_LIBVIRT_BLOCK_LM_WITH_VOLUMES_VERSION = (1, 2, 17)
 # libvirt discard feature
-MIN_QEMU_DISCARD_VERSION = (1, 6, 0)
+MIN_QEMU_DISCARD_VERSION = (1, 5, 3)
 # While earlier versions could support NUMA reporting and
 # NUMA placement, not until 1.2.7 was there the ability
 # to pin guest nodes to host nodes, so mandate that. Without
