--- /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py	2016-10-13 14:01:30.000000000 +0200
+++ /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py_fixed	2017-02-14 13:40:10.090947306 +0100
@@ -426,7 +426,7 @@
 MIN_LIBVIRT_BLOCK_LM_WITH_VOLUMES_VERSION = (1, 2, 17)
 # libvirt discard feature
 MIN_LIBVIRT_DISCARD_VERSION = (1, 0, 6)
-MIN_QEMU_DISCARD_VERSION = (1, 6, 0)
+MIN_QEMU_DISCARD_VERSION = (1, 5, 3)
 # While earlier versions could support NUMA reporting and
 # NUMA placement, not until 1.2.7 was there the ability
 # to pin guest nodes to host nodes, so mandate that. Without
