diff --git a/usr/lib/python2.7/site-packages/keystone/federation/utils.py b/tmp/utils.py
index dc74c9e..7a4f414 100644
--- a/usr/lib/python2.7/site-packages/keystone/federation/utils.py
+++ b/tmp/utils.py
@@ -383,8 +383,8 @@ def transform_to_group_ids(group_names, mapping_id,
                 group['name'], resolve_domain(group['domain']))
             yield group_dict['id']
         except exception.GroupNotFound:
-            raise exception.MappedGroupNotFound(group_id=group['name'],
-                                                mapping_id=mapping_id)
+            LOG.debug('Skip mapping group %s; has no entry in the backend',
+                      group['name'])
 
 
 def get_assertion_params_from_env(request):
